client/app/private-database/state/STATE.html

```diff
@@ -29,10 +29,16 @@ <h3 class="oui-header__title" data-i18n-static="privateDatabase_dashboard_genera
                                     <div class="d-flex w-100">
                                         <div>
                                             <strong data-i18n-static="privateDatabase_dashboard_ram"></strong>
-                                            <span data-ng-bind="::stateCtrl.database.ram.value + i18n['unit_size_' + stateCtrl.database.ram.unit]"></span>
-                                            <button type="button" class="btn btn-icon" data-ng-click="stateCtrl.goToOom()">
-                                                <i class="fa fa-info-circle" aria-hidden="true"></i>
+                                            <button class="btn btn-xs" type="button"
+                                                    data-ng-class="stateCtrl.database.oom.realList.length >= stateCtrl.database.oom.nbOomError ? 'btn-danger' : (stateCtrl.database.oom.realList.length > 0 ? 'btn-warning' : 'btn-success')"
+                                                    data-ng-click="stateCtrl.goToOom()">
+                                                <span data-ng-bind="::stateCtrl.database.ram.value + i18n['unit_size_' + stateCtrl.database.ram.unit]"></span>
                                             </button>
+                                            <p class="small" data-ng-class="stateCtrl.database.oom.realList.length >= stateCtrl.database.oom.nbOomError ? 'text-danger' : (stateCtrl.database.oom.realList.length > 0 ? 'text-warning' : '')">
+                                                <i data-ng-if="stateCtrl.database.oom.realList.length > 0" class="fa fa-exclamation-triangle" aria-hidden="true"></i>
```

Comment:
Don't use the `<i>` anchor. Use `<strong>` if the sentence is important and use `<em>` to stress elements that can change the meaning of a sentence